fixtures/flows/brief/specialist.js

```diff
@@ -1,124 +1,129 @@
-export const create = async (params) => {
-    console.log(`Starting to create ${params.areaOfExpertise} brief`);
-    await createBrief();
-    await fillRole(params.title);
-    await selectLocation(params.locations);
-    await fillDescriptionOfWork();
-    await fillEvaluationProcess(params.areaOfExpertise, params.evaluations);
-    await fillHowLong();
-    await fillQuestionAnswer();
-    await fillWhoCanRespond();
-    await publishBrief();
+import * as util from '../../flows/utils'
+
+/* eslint-disable no-await-in-loop */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`